Code/autopkg

```diff
@@ -806,6 +807,9 @@ def repo_delete(argv):
             recipe_search_dirs.remove(repo_path)
         # now remove the repo files
         try:
+            if is_windows():
+                # we need to remove readonly, hidden and system bits
+                os.system("attrib.exe -r -h -s " + repo_path + "\\*.* /S /D")
```

Comment:
This seems to be to fix a bug that is unrelated to fixing xattr. What is the issue this is solving? Is there a pythonic way to solve this using Pathlib or similar, rather than calling `attrib.exe`?

I use autopkg a ton on windows, and I have never hit the problem that this seems to be fixing. What autopkg command causes this problem?

Reply:
This changes came into this PR, after i issued it.
Was not intend to do so...

Reply:
ah, I see. That is a pain. Once you create the PR from a branch, if you make any further changes to that branch, then it gets included. 

It is best to create a custom branch only for the purposes of the PR, which is a bit annoying.